vernac/mltop.ml

```diff
@@ -334,23 +333,38 @@ let _ =
 
 (* Liboject entries of declared ML Modules *)
 
+(* Digest of module used to compile the file *)
+type ml_module_digest =
+  | NoDigest
+  | AnyDigest of Digest.t             (* digest of any used cma / cmxa *)
+
 type ml_module_object = {
   mlocal : Vernacexpr.locality_flag;
-  mnames : string list
+  mnames : (string * ml_module_digest) list
```

Comment:
The name of this field is now wrong.

Reply:
I didn't want to change it to minimize diff noise, but if you think it is worth it I'll tweak.

Reply:
Maybe we should just have an extra field `mdigests : digest list`?

Reply:
Seems rather the module info should go in a single list, unless you want to deprecate the capability of multi-load of `Declare ML Module`.